iam/machine/machine.tf

```diff
@@ -0,0 +1,47 @@
+# ------------------------------------------------------------------------------
+# Variables
+# ------------------------------------------------------------------------------
+variable "name" {
```

Comment:
I've used `prefix` in all other modules, and I think it'd be nice to be consistent with the variables - usually I just copy the description and everything from an existing module for that reason.